packages/api/src/StyleOptions.ts

```diff
@@ -928,6 +928,18 @@ type StyleOptions = {
    * New in 4.19.0.
    */
   codeBlockTheme?: 'github-light-default' | 'github-dark-default';
+
+  /**
+   * Feedback buttons placement
+   *
+   * - `'activity-actions'` - place feedback buttons inside activity actions
+   * - `'activity-status'` - place feedback buttons inside activity status
+   *
+   * @default 'activity-status'
+   *
+   * New in 4.19.0.
+   */
+  feedbackActionsPlacement?: 'activity-actions' | 'activity-status';
```

Comment:
If we have a better activity status middleware and activity actions middleware, and exposing the `<Feedback>` component, web devs can move the feedback buttons by themselves.

I am thinking, web devs who need to move the placement are mostly skin developers. And we didn't test "activity actions" in white-label.

What do you think?

If you agree on this, for time constraint, we could mark this as experimental.

Reply:
I still think the default placement in `styleOptions` makes sense as we already support vote actions? But I'm with you. Marking as experimental for now as we don't lose anything this way.